### PR TITLE
Add Windscribe

### PR DESCRIPTION
**Pull request by dngray**
Progression of the discussion at https://github.com/privacyguides/privacyguides.org/discussions/372#discussioncomment-1691190

Requires: https://github.com/privacyguides/privacyguides.org/pull/1311 to be merged first.

### CODE REVIEW

**Comment by blacklight447** [comment]
Does windscribe pass our criteria?

**Comment by dngray** [comment]
> Does windscribe pass our criteria?

Yes.

**Comment by blacklight447** [comment]
Just checked it all, seems to be good to go.

File: docs/vpn.en.md [hunk 1]
```diff
@@ -202,6 +202,55 @@ Find a no-logging VPN operator who isn’t out to sell or read your web traffic.
...
+
+??? check annotate "63 Countries"
+
+    Windscribe has [servers in 63 countries](https://windscribe.com/features/large-network) (1). Picking a VPN provider with a server nearest to you will reduce latency of the network traffic you send. This is because of a shorter route (less hops) to the destination.
+
+    We also think it's better for the security of the VPN provider's private keys if they use [dedicated servers](https://en.wikipedia.org/wiki/Dedicated_hosting_service), instead of cheaper shared solutions (with other customers) such as [virtual private servers](https://en.wikipedia.org/wiki/Virtual_private_server).
+
+1. As of 2022/05/17
+
+??? check "Independently Audited"
```
Comment (Dyrimon) [inline docs/vpn.en.md:227]: There should be a link to the audit report and details of the audit

File: docs/vpn.en.md [hunk 2]
```diff
@@ -202,6 +202,55 @@ Find a no-logging VPN operator who isn’t out to sell or read your web traffic.
...
+
+??? check "Open Source Clients"
+
+    As of May 2022 [Windscribe applications are now open source](). Source code can be obtained from their [GitHub organization](https://github.com/Windscribe).
+
+??? check "Accepts Cash and Monero"
+
+    In addition to accepting credit/debit cards and PayPal, Windscribe accepts Bitcoin as payment.
```
Comment (Dyrimon) [inline docs/vpn.en.md:235]: Description doesn't explain if they accept Cash/Moneo or not

File: docs/vpn.en.md [hunk 3]
```diff
@@ -202,6 +202,55 @@ Find a no-logging VPN operator who isn’t out to sell or read your web traffic.
...
+??? check "Remote Port Forwarding"
+
+    Remote ephemeral [port forwarding](https://en.wikipedia.org/wiki/Port_forwarding) is possible with a [Pro plan](https://windscribe.com/features/port-forwarding). For a permanent port forward, you need to purchase a [static IP](https://windscribe.com/staticips).
+
+??? check "Mobile Clients"
+
+    In addition to providing standard OpenVPN configuration files, Windscribe has mobile clients for [App Store](https://apps.apple.com/app/id1129435228) and [Google Play](https://play.google.com/store/apps/details?id=com.windscribe.vpn) allowing for easy connections to their servers.
+
+??? info "Additional Functionality"
+
```
Comment (Dyrimon) [inline docs/vpn.en.md:252]: I guess there are no additional functionality? Maybe mention R.O.B.E.R.T

**Review by Dyrimon — CHANGES_REQUESTED** [review]
I've left my opinions on comments

**Comment by dngray** [comment]
Yes, this one is an early draft, nowhere near finished.

**Comment by EdwardLangdon** [comment]
I think @yegors could help us out here.

File: docs/vpn.en.md [hunk 4]
```diff
@@ -202,6 +201,53 @@ Find a no-logging VPN operator who isn’t out to sell or read your web traffic.
...
+    **Free** - **Yearly Plan USD $48.96/year**
+
+    [:octicons-home-16: Homepage](https://windscribe.com){ .md-button .md-button--primary }
+
+??? check "69 Countries"
+
+    Windscribe has [servers in 69 countries](https://windscribe.com/features/large-network), as of 2022/06/01. Picking a VPN provider with a server nearest to you will reduce latency of the network traffic you send. This is because of a shorter route (less hops) to the destination.
+
+    We also think it's better for the security of the VPN provider's private keys if they use [dedicated servers](https://en.wikipedia.org/wiki/Dedicated_hosting_service), instead of cheaper shared solutions (with other customers) such as [virtual private servers](https://en.wikipedia.org/wiki/Virtual_private_server).
```
Comment (jonaharagon) [inline docs/vpn.en.md:221]: Why do we say this in the countries admonition of every provider? It doesn't even make sense in this context?

Comment (dngray) [reply docs/vpn.en.md:221]: It was something that was always there. Not sure it makes a lot of sense no.

**Review by purtual — CHANGES_REQUESTED** [review]

**Comment by yegors** [comment]
Just FYI, Windscribe is now running on a RAM-disk based infrastructure. We silently finished the deployment of this last week to  99% of servers.  

Expect a detailed writeup on how it works in the coming weeks. Audit has been scheduled for end of November by Cure53 (earliest they could do).

**Comment by jonaharagon** [comment]
@yegors last update we've heard was that an audit was scheduled for Nov 2022, and before that it was June 2022; has any progress been made on this front?

https://rus.windscribe.com/knowledge-base/articles/has-windscribe-been-audited says your desktop and mobile apps have been audited, which would be great if those were published.

**Comment by yegors** [comment]
@jonaharagon We did indeed conduct a **pre-production** audit of our new infrastructure in Dec of 2022. As this stack is not yet in production, publishing it is pointless. Once 2.0 stack is actually deployed, the results of this audit and a new production audit will be published. 

Our Android source code has been made public as well: https://github.com/Windscribe/Android-App

iOS will follow in a several weeks. 

The code audits for Desktop, iOS and Android were done a while ago, ahead of open sourcing. We may link them in the relevant repos soon.

**Comment by jonaharagon** [comment]
Thanks @yegors, good to hear! I think we are *most* concerned about the code audits for your clients, which it sounds like are already completed anyways? When those are published I would be fine with merging this PR and adding Windscribe to the site, and then publishing your infrastructure updates/audits as a future update.

**Comment by jermanuts** [comment]
Thanks @yegors for listening to PG's feedback. I hope you [fix](https://github.com/Windscribe/Desktop-App) mirroring bug. Also, there are 2 protocols (WStunnel and Stealth) are they open source?

**Comment by yegors** [comment]
> Thanks @yegors for listening to PG's feedback. I hope you [fix](https://github.com/Windscribe/Desktop-App) mirroring bug. Also, there are 2 protocols (WStunnel and Stealth) are they open source?

What mirroring bug are you referring to? The fact that Github was behind what's actually released? yeah, we addressed that. 

As for WStunnel/Stunnel, everything is open sourced, including those 2 projects themselves

https://github.com/erebe/wstunnel
https://www.stunnel.org/

**Comment by freddy-m** [comment]
What is the status of this @dngray @privacyguides/team?

**Comment by jonaharagon** [comment]
This is going on 2 years this weekend. Closing since nothing has changed, we can reopen if it's ever actually ready.

**Comment by IkelAtomig** [comment]
https://blog.windscribe.com/freshscribe-next-generation-vpn-infrastructure-2/ - Time to reopen, I guess.

**Pull request closed without merging**